src/commands/convert.ts

```diff
@@ -0,0 +1,56 @@
+import { promises as fPromises } from 'fs';
+import { flags } from '@oclif/command';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { load } from '../models/SpecificationFile';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+import { convert } from '@asyncapi/converter';
+
+export default class Convert extends Command {
+  static description = 'convert asyncapi documents older to newer versions';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    file: flags.string({ char: 'f', description: 'path to the file to convert' }),
+    output: flags.string({ char: 'o', description: 'path to the file where the result is saved' }),
+    'target-version': flags.string({ char: 't', description: 'asyncapi version to convert to', default: '2.3.0' })
+  }
+
+  static args = [
+    { name: 'spec-file', description: 'spec path, url, or context-name', required: false },
+  ]
+
+  async run() {
+    const { args, flags } = this.parse(Convert);
+    const filePath = args['spec-file'];
+    let specFile;
+    let convertedFile;
+
+    try {
+      // LOAD FILE
+      specFile = await load(filePath);
+
+      // CONVERSION
+      convertedFile = await convert(specFile.text(), flags['target-version'], {});
+      if (convertedFile) {
+        this.log(`File ${specFile.getFilePath()} successfully converted!`);
+      } else {
+        return;
```

Comment:
Please add here some log, why we stop functionality.

Reply:
```suggestion
        this.log(`File ${specFile.getFilePath()} could not be converted.`);
        return;
```

Reply:
Yeah, but awesome would be if we will describe why we cannot convert, you know, not supported version etc.

Reply:
Taking into account the converter already throws a message when the conversion is not supported, which one among these options do you think should be more suitable? 🤔 

1. A generic message like:
`Conversion to target-version provided is not supported.`

2. A more specific message like:
`Conversions to lower, equal or non-existing target-version are not supported. `

These are the converter's messages I'm referring to:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/81691177/154379100-f8c97c39-4420-485a-a13c-1e811c0219ec.png">

Reply:
@peter-rr 

Could you wait for https://github.com/asyncapi/converter-js/pull/99 PR? When we will merge it, we will be able to print the message from error like:

```ts
this.log(e.message);
```

Reply:
And then (of course) we should wrap code with `try catch`:

```ts
try {
  convertedFile = await convert(specFile.text(), flags['target-version'], {});
} catch(e) {
  this.log(e.message);
}
```

Reply:
Sure, I'll wait for it! Now I understand why the error cases from the converter were not handled by the catch block 😅 Thanks!

Reply:
@peter-rr Ok you can use `0.8.0` version of converter where we throw errors. Please update dep and propagate my suggestion about `try {} catch {}` and I guess that we can open PR to the final review, what do you think? :)

Reply:
Yeah, sounds great 👌 Changes suggested already applied and opening PR for the final review 💪 
Tests are still missing though.